Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -285,7 +285,7 @@ extension NSManagedObjectContext {
         // for blocked 1:1 channels on channel list query
         if let isBlocked = payload.isBlocked {
             dto.isBlocked = isBlocked
-        } else {
+        } else if query != nil {
```

Comment:
is this condition enough? I think it's better that this is only solved on the backend.